LibreOfficeWriter_Field.au3

```diff
@@ -8,23 +8,12 @@
 #include "LibreOfficeWriter_Doc.au3"
 
 ; #INDEX# =======================================================================================================================
-; Title .........: Libre Office Writer (LOWriter)
+; Title .........: LibreOffice UDF
 ; AutoIt Version : v3.3.16.1
-; UDF Version    : 0.0.0.3
-; Description ...: Provides basic functionality through Autoit for interacting with Libre Office Writer.
+; Description ...: Provides basic functionality through AutoIt for Inserting and Modifying L.O. Writer Fields.
```

Comment:
Fields ?

Reply:
Yes, I mean like this:
https://help.libreoffice.org/latest/en-US/text/swriter/guide/fields.html

Reply:
My mistake. I was looking wrong place.
![image](https://github.com/mlipok/Au3LibreOffice/assets/11089482/e7f1225b-30a2-41ab-9869-0bb8ebcfa6a5)

Reply:
Oh Okay. I understand now.